src/components/afm/tests/Heatmap.spec.tsx

```diff
@@ -27,7 +27,7 @@ describe("Heatmap", () => {
             />,
         );
 
-        return testUtils.delay().then(() => {
+        await testUtils.delay().then(() => {
```

Comment:
I'm not sure what you were trying to achieve... but consider something like this:
```
 await testUtils.delay();
        wrapper.update();
        const dimensions = wrapper.find(CoreHeatmap).props().resultSpec.dimensions;
        expect(dimensions).toEqual([{ itemIdentifiers: ["measureGroup"] }, { itemIdentifiers: ["heat"] }]);
```